lib/resolvers/embedded.resolver.ts

```diff
@@ -87,6 +87,41 @@ export function createResolverForEmbedded(definition: Definition, field: string)
       const parent = await this.model.findById(parentId).select(field);
       return parent[field].map((item: any) => appendIdAndTransform(embeddedDefinition, item)) as any;
     }
+
+    @Mutation(() => [Typer.getObjectType(embeddedDefinition)])
+    async [`update${util.toPascalCase(definition.name)}${util.toPascalCase(field)}`](
+      @Args(
+        'input',
+        { type: () => [Typer.getUpdateInputType(embeddedDefinition)] },
+        new ValidationPipe({
+          transform: true,
+          expectedType: Typer.getUpdateInputType(embeddedDefinition),
+        }),
+      )
+      input: any,
+      @Args(`${util.toCamelCase(definition.name)}Id`, {
+        type: () => graphql.GraphQLID,
+      })
+      parentId: string,
+    ): Promise<T[]> {
+      const parent = await this.model.findById(parentId);
+      if (!parent) {
```

Comment:
code style little thing

```suggestion
      if (util.isNil(parent)) {
```